### DISCUSSION TITLE
Unable to access code-server on assigned port?

### DISCUSSION THREAD

**Discussion by broomwalker** [coder/code-server#6176] [Q&A] [answered]
Hi all,

I’ve tried the code-server docker-compose and Portainer App Template. 

I’m having issues accessing the website through the assigned port - 3443.

                                                                         
```
version: "2.1"
services:
��code-server:
����image: lscr.io/linuxserver/code-server:latest
����container_name: code-server
����environment:
������- PUID=1000
������- PGID=1000
������- TZ=Australia/Sydney
������- PASSWORD=code-serverpassword #optional
������- SUDO_PASSWORD=sudopassword #optional
������- DEFAULT_WORKSPACE=/config/workspace #optional
����volumes:
������- /path/to/appdata/config:/config
����ports:
������- 3443:3443
����restart: unless-stopped
```

I’ve checked my logs in Portainer and weirdly:

- It keeps defining back to 8443? This would clash with ngenx
- PGID doesn’t align with docker-compose?
- https://hub.docker.com/r/linuxserver/code-server Makes mention that it should be http://<your-ip>:3443, but the logs show 0.0.0.0:8443?

> [migrations] started
> [migrations] no migrations found
> ───────────────────────────────────────
>       ██╗     ███████╗██╗ ██████╗ 
>       ██║     ██╔════╝██║██╔═══██╗
>       ██║     ███████╗██║██║   ██║
>       ██║     ╚════██║██║██║   ██║
>       ███████╗███████║██║╚██████╔╝
>       ╚══════╝╚══════╝╚═╝ ╚═════╝ 
>    Brought to you by linuxserver.io
> ───────────────────────────────────────
> To support LSIO projects visit:
> https://www.linuxserver.io/donate/
> ───────────────────────────────────────
> GID/UID
> ───────────────────────────────────────
> User UID:    1000
> User GID:    911
> ───────────────────────────────────────
> setting up sudo access
> adding abc to sudoers
> setting sudo password using SUDO_PASSWORD env var
> New password: Retype new password: passwd: password updated successfully
> [custom-init] No custom files found, skipping...
> [2023-04-30T10:30:33.064Z] info  code-server 4.12.0 1da7cda39e54faa087cf129f0f85d4d4e63e81b8
> [2023-04-30T10:30:33.067Z] info  Using user-data-dir ~/data
> [2023-04-30T10:30:33.117Z] info  Using config file ~/.config/code-server/config.yaml
> [2023-04-30T10:30:33.118Z] info  HTTP server listening on http://0.0.0.0:8443/
> [2023-04-30T10:30:33.118Z] info    - Authentication is enabled
> [2023-04-30T10:30:33.118Z] info      - Using password from $PASSWORD
> [2023-04-30T10:30:33.118Z] info    - Not serving HTTPS
> [ls.io-init] done.

**Comment by code-asher** [comment, marked as answer]
> https://hub.docker.com/r/linuxserver/code-server Makes mention that it should be http://:3443

I checked the link but I was not able to find any mention of 3443.  Looks like it is supposed to be 8443?

> ����ports:
> ������- 3443:3443

You could probably just use `3443:8443`.  Alternatively you could pass `--port 3443` to code-server.

I am not sure about PGID though; `linuxserver/code-server` is third-party so you might have better luck asking them for container-specific questions: https://github.com/linuxserver/docker-code-server

**Comment by broomwalker** [comment]
3443:8443 did the trick!

Thank you!
